logic/atrium-logic-jvm/src/main/kotlin/ch/tutteli/atrium/logic/impl/DefaultPathAssertions.kt

```diff
@@ -98,13 +98,16 @@ class DefaultPathAssertions : PathAssertions {
     }.let(block)
 
     override fun <T : Path> isReadable(container: AssertionContainer<T>): Assertion =
-        filePermissionAssertion(container, READABLE, AccessMode.READ)
+        filePermissionAssertion(container, READABLE, AccessMode.READ, DescriptionBasic.IS)
+
+    override fun <T : Path> isNotReadable(container: AssertionContainer<T>): Assertion =
+        filePermissionAssertion(container, NOT_READABLE, AccessMode.READ, DescriptionBasic.IS_NOT)
```

Comment:
you also need a flag for the `not` part. This way you still check if it is readable, you only change the meaning of the shown text